management/prompt_test.go

```diff
@@ -17,35 +18,73 @@ func TestPrompt(t *testing.T) {
 		}
 	})
 
-	// update to the new identifier first experience
-	err := m.Prompt.Update(&Prompt{
-		UniversalLoginExperience: "new",
-		IdentifierFirst:          auth0.Bool(true),
+	t.Run("Update to the new identifier first experience", func(t *testing.T) {
+		err := m.Prompt.Update(&Prompt{
+			UniversalLoginExperience: "new",
+			IdentifierFirst:          auth0.Bool(true),
+		})
+		if err != nil {
+			t.Error(err)
+		}
+
+		ps, err := m.Prompt.Read()
+		if err != nil {
+			t.Error(err)
+		}
+		expect.Expect(t, ps.UniversalLoginExperience, "new")
+		expect.Expect(t, ps.IdentifierFirst, auth0.Bool(true))
 	})
-	if err != nil {
-		t.Error(err)
-	}
-
-	ps, err := m.Prompt.Read()
-	if err != nil {
-		t.Error(err)
-	}
-	expect.Expect(t, ps.UniversalLoginExperience, "new")
-	expect.Expect(t, ps.IdentifierFirst, auth0.Bool(true))
-
-	// update to the classic non identifier first experience
-	err = m.Prompt.Update(&Prompt{
-		UniversalLoginExperience: "classic",
-		IdentifierFirst:          auth0.Bool(false),
+
+	t.Run("Update to the classic non identifier first experience", func(t *testing.T) {
+		err := m.Prompt.Update(&Prompt{
+			UniversalLoginExperience: "classic",
+			IdentifierFirst:          auth0.Bool(false),
+		})
+		if err != nil {
+			t.Error(err)
+		}
+
+		ps, err := m.Prompt.Read()
+		if err != nil {
+			t.Error(err)
+		}
+		expect.Expect(t, ps.UniversalLoginExperience, "classic")
+		expect.Expect(t, ps.IdentifierFirst, auth0.Bool(false))
+	})
+}
+
+func TestPromptCustomText(t *testing.T) {
+	t.Cleanup(func() {
+		prompt := "login"
+		lang := "en"
+		body := "{}"
+
+		err := m.Prompt.SetCustomText(prompt, lang, &body)
+		if err != nil {
+			t.Error(err)
+		}
+	})
+
+	t.Run("Retrieve custom text", func(t *testing.T) {
```

Comment:
Please correct me if I am wrong but it doesn't seem that we are asserting that the custom text is the one we expect over here.

Reply:
Since these are integration tests, the goal of the test is not to assert on the API response (that's not the SUT), but that the request completes successfully. There is a `t.Error(err)` for the error case, otherwise it succeeds.
However, if you think we should assert on the API response, I'll add it.